mailpoet/tests/integration/API/JSON/APITest.php

```diff
@@ -73,11 +73,15 @@ function($api) use (&$called) {
         expect($api instanceof JSONAPI)->true();
       }
     );
+    $wpStub = Stub::make(new WPFunctions, [
+      'wpVerifyNonce' => asCallable(function() {
+        return true;
+      })]);
     $api = Stub::makeEmptyExcept(
```

Comment:
The only reason this test was passing previously is that the check at http://github.com/mailpoet/mailpoet/blob/10e53ccd1be3ce3b83d287655bf553feb9e524c1/mailpoet/lib/API/JSON/API.php#L108-L108 was checking for false, and the empty stub for `checkToken` was returning `null`.